### ISSUE TITLE
3.10.2 no avatars issue

### ISSUE THREAD

**Issue by cantonduck** [siacs/Conversations#1428]
Hi,

just managed to install 3.10.2 in my synology NAS, came from 3.9.3.
Same issue like before (https://github.com/siacs/Conversations/issues/1389): conversations client pusblishes avatar, pidgin client, does not get/see avatar.
whereas pidgin client publishes avatar, but conversations client sees the pidgin avatar...

also tried this one: https://el-tramo.be/blog/vcardextpres/  but no success

any ideas?

**Comment by iNPUTmice** [comment]
I think pidgin does not support PEP avatars. Whereas Conversations support vcard avatars in a read only mode. (On top of that PEP support might still be broken in OpenFire I haven't checked)

**Comment by cantonduck** [comment]
In some other postings you and another user were discussing that and it's veen told that this problem was corrected in 3.10.0... maybe there is some other mistake that I did when setting up the server? I am using 3.10.2 with mysql and the internal DB setting. Should I have used the LDAP or 3rd option when installing?


Am 22. September 2015 09:48:15 MESZ, schrieb Daniel Gultsch <notifications@github.com>:
